wireapp/wire-android
# feat: confirmation dialog for sending ping messages (WPB-2898)

<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/WPB-2898" title="WPB-2898" target="_blank"><img alt="Story" src="https://wearezeta.atlassian.net/rest/api/2/universal_avatar/view/type/issuetype/avatar/10815?size=medium" />WPB-2898</a>  [Android] confirmation dialog for sending ping messages
  </td></table>
  <br />
 








----
#### PR Submission Checklist for internal contributors

- The **PR Title**

- The **PR Description**
----

# What's new in this PR?

### Issues

We need to show a confirmation dialog when someone tries to ping a group that has more than 3 participants, mainly to prevent accidental clicks

Needs releases with:


### Testing

#### Test Coverage (Optional)


#### How to Test

On a group that has more than 3 participants try to ping.
You should see a confirmation dialog asking you to confirm the ping

![Screenshot_20240812_110203](https://github.com/user-attachments/assets/8d1aa703-4695-4970-be40-e18940324e06)

----
#### PR Post Submission Checklist for internal contributors (Optional)

----
#### PR Post Merge Checklist for internal contributors

----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.
